### PR TITLE
[#IOPID-106] Add retry info to `HandlePubKeyRevoke` exception

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes

Add `retryCount` and `maxRetryCount` to exception.
Remove sampling.

#### Motivation and Context

Enable adding an alert when all retries fail

#### How Has This Been Tested?

Unit test

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: HandlePubKeyRevoke/handler.ts [hunk 1]
```diff
@@ -164,14 +164,22 @@ export const handleRevoke = (
...
+          ),
+          maxRetryCount: String(
+            context.executionContext.retryContext?.maxRetryCount ?? "undefined"
+          ),
           modelId: err.modelId ?? "",
           name: "lollipop.pubKeys.revoke.failure"
+          /* eslint-enable sort-keys */
         },
-        tagOverrides: { samplingEnabled: String(isTransient) }
+        tagOverrides: { samplingEnabled: "false" }
```
Comment (gquadrati) [inline HandlePubKeyRevoke/handler.ts:182]: Remove sampling for testing queries. We can enable it again after tests are complited.

File: HandlePubKeyRevoke/handler.ts [hunk 2]
```diff
@@ -164,14 +164,22 @@ export const handleRevoke = (
       trackException(telemetryClient, {
         exception: new Error(error),
         properties: {
+          /* eslint-disable sort-keys */
```
Comment (BurnedMarshal) [inline HandlePubKeyRevoke/handler.ts:167]: Can we sort properties? Is `sort-keys` disabled for some reason?

**Review by BurnedMarshal — APPROVED** [review]
LGTM apart a minor comment.

File: HandlePubKeyRevoke/handler.ts (hunk 2 above)
Comment (gquadrati) [reply HandlePubKeyRevoke/handler.ts:167]: Just because I don't like it  so much :) I'll fix it

Comment (gquadrati) [reply HandlePubKeyRevoke/handler.ts]: Done in [21a7159](https://github.com/pagopa/io-functions-lollipop/pull/37/commits/21a71591e53b5c92d31933e725e786c329bf146d)

**Pull request merged**